src/focus.service.ts

```diff
@@ -625,6 +622,10 @@ export class FocusService {
       || !!record;
   }
 
+  private isVisible(el: HTMLElement) {
+    return !!(el.offsetWidth || el.offsetHeight || el.getClientRects().length);
```

Comment:
Bear in mind that this is not exactly equivalent

```js
var el = document.createElement('div');
document.body.appendChild(el);
el.style.width = el.style.height = '10px';
el.style.visibility = 'hidden';
console.log(!!(el.offsetWidth || el.offsetHeight || el.getClientRects().length))
```

The change looks fine but make sure you don't get tripped up